From ad611a93d4ffda51ec620c9b17382654b37d475e Mon Sep 17 00:00:00 2001
From: Marquise Stein <botocratic@gmail.com>
Date: Sat, 7 Nov 2020 12:53:04 -0800
Subject: [PATCH] Fix: Patch keccak to prevent node 14 segfault

It does this by never loading the native module in the first place.

See:

    https://github.com/trufflesuite/ganache-core/commit/c26ba24ebf24d1c09b328a76baf0cf43e82fba04
---
 index.js | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/index.js b/index.js
index 24fb71a..bb62b01 100644
--- a/index.js
+++ b/index.js
@@ -1,5 +1 @@
-try {
-  module.exports = require('./bindings')
-} catch (err) {
-  module.exports = require('./js')
-}
+module.exports = require('./js')
-- 
2.17.1

